### PR TITLE
DCAPT-19 improve css selector for PR merge submit button

### PR DESCRIPTION
**Pull request by ukuhnhardt**
See https://ecosystem.atlassian.net/browse/DAPT-19

### CODE REVIEW

File: app/selenium_ui/bitbucket/pages/selectors.py [hunk 1]
```diff
@@ -193,7 +193,7 @@ class PullRequestLocator:
     merge_spinner = (By.CSS_SELECTOR, "aui-spinner[size='small']")
     diagram_selector = {'6': (By.CSS_SELECTOR, 'div.diagram-image'), '7': (By.CLASS_NAME, 'branches-diagram')}
     pull_request_modal_merge_button = {'6': (By.CSS_SELECTOR, 'button.confirm-button'),
-                                       '7': (By.CSS_SELECTOR, "button[type='submit']")}
+                                       '7': (By.CSS_SELECTOR, ".merge-dialog button[type='submit']")}
```
Comment (ukuhnhardt) [inline app/selenium_ui/bitbucket/pages/selectors.py:196]: ideally bitbucket core team would add a `data-test-id` attr to the PR merge modal.

**Comment by ometelytsia** [comment]
Hi @ukuhnhardt, thank you for your contribution! Could you pls close this PR and raise a new one to the `dev` branch. `master` branch gets changes only during the releases. Thank you.

**Comment by ukuhnhardt** [comment]
> Hi @ukuhnhardt, thank you for your contribution! Could you pls close this PR and raise a new one to the `dev` branch. `master` branch gets changes only during the releases. Thank you.

didn't have to close it - just changed the 'base' branch. Hope that's ok.

**Comment by ometelytsia** [comment]
> > Hi @ukuhnhardt, thank you for your contribution! Could you pls close this PR and raise a new one to the `dev` branch. `master` branch gets changes only during the releases. Thank you.
> 
> didn't have to close it - just changed the 'base' branch. Hope that's ok.

Hi @ukuhnhardt. Just changing the branch is **not** OK. As you can see now your PR contains many not-needed commits, because you've branched not from the latest `dev`.

**Comment by ukuhnhardt** [comment]
created #610 to target `dev` branch

**Pull request closed without merging**